src/systemdhook.c

```diff
@@ -132,6 +132,10 @@ static int chcon(const char *path, const char *label) {
 			if (setfilecon (full_path, label) < 0) {
 				pr_perror("Failed to set context %s on %s", label, path);
 			}
+                        /* Change uid and gid to something the container can handle */
+                        if (chown(full_path, uid, gid) == -1) {
+                                pr_perror("Failed to set full_path owner: %s, errno: [%d]", full_path, errno);
```

Comment:
Add UID and GID to the error.
(failed to chown %d;%d to full_path: %s:%s, uid, gid, full_path, errno)